### DISCUSSION TITLE
peak to regulons association

### DISCUSSION THREAD

**Discussion by adrdufour** [aertslab/scenicplus#130] [Q&A]
Hello,

I would like to know how the peak is associated with the regulons. I have run the pipeline with few changes and observed slightly different results in a peak-to-gene association. I also observed a lot of regulons that are positive to negative “+ -”.

As you can see here, the two regulons ET and TR have a great expression of their Transcription factor and gene expression of the regulons is significantly expressed in the corresponding clusters (better result than in classical SCENIC approach), but the corresponding ATAC regions are reverse of those result. For some other regulons, the result seems correct, like NANOG.

![selected_regulons](https://user-images.githubusercontent.com/16325905/225590012-c82a8be0-17ea-4d68-958e-53d4e8274709.png)

As I have a custom species, I would like to know if I need to change my annotation and motif annotation or if there are parameters that can influence those assignations.

Thanks for your helps,
Adrien Dufour.

**Comment by SeppeDeWinter** [comment]
Hi @Goultard59 

Region-to-gene relationships are calculated using random forrest regression, predicting gene expression from region accessibility for all regions within 150kb up-and-downstream of the gene including introns (by default). The Feature importance score is used as the region-to-gene score. This is done separately for regions for which the accessibility correlates positively and negatively with the target's gene expression. 

The second sign denominates wether the region correlates positively or negatively with the target gene. (the first is about the TF-to-gene correlation). In your case this is negative, so you would expect that that the region accessibility is anti-correlated to the gene expression (as you show in your plot). This has nothing to do with your annotation. Do you also have eRegulons with sign `+_+_` ? I would suggest to focus on those.

Best,

Seppe

**Comment by adrdufour** [comment]
Hi,

Thanks for the clarification. After going further into the details between the different runs, I see that the difference was the peak detection and that there are disappearance peaks associated with gene expression. So I was wondering if I need to split the cluster at different time points (Cluster1_E7, Cluster1_E9, Cluster1_E11) for now I am just providing cluster information (Cluster1, Cluster2...).

I also would like to know where the information on the motif associated with the regulons is stored.

Thank you for your responses,
Adrien.
